camel/configs/__init__.py

```diff
@@ -32,6 +33,8 @@
     'OPENAI_API_PARAMS',
     'AnthropicConfig',
     'ANTHROPIC_API_PARAMS',
+    'GROQ_LLAMA3_API_PARAMS',
```

Comment:
groq also support other model types beyond llama3, we can make this integration more general

Reply:
Groq supports some famous open source models, but the configs of every model may not be the same. After we have integrated other models served by groq, we can improve this at that time. What do you think of it?